config/development/crd/bases/operator.kyma-project.io_telemetries.yaml

```diff
@@ -49,6 +49,37 @@ spec:
           spec:
             description: TelemetrySpec defines the desired state of Telemetry
             properties:
+              log:
+                description: LogSpec defines the behavior of the log gateway
+                properties:
+                  gateway:
+                    properties:
+                      scaling:
+                        description: Scaling defines which strategy is used for scaling
+                          the gateway, with detailed configuration options for each
+                          strategy type.
+                        properties:
+                          static:
+                            description: |-
+                              Static is a scaling strategy enabling you to define a custom amount of replicas to be used for the gateway. Present only if Type =
+                              StaticScalingStrategyType.
+                            properties:
+                              replicas:
+                                description: Replicas defines a static number of pods
```

Comment:
```suggestion
                                description: Replicas defines a static number of Pods
```

Reply:
Would it be fine if I do this in a separate PR? Since this one got really big 😅

Reply:
As discussed, ignore the consistency across the whole project. It's fine if we simply try to be consistent in the doc.